NOTES-NONSTOP.md

```diff
@@ -0,0 +1,148 @@
+NOTES FOR THE HPE NONSTOP PLATFORM
+==============================
+
+Requirement details
+-------------------
+
+ In addition to the requirements and instructions listed
+ in [INSTALL.md](INSTALL.md), the following are required as well:
+
+  * The TNS/X platform supports hardware randomization. Specify
+    the `--with-rand-seed=rdcpu` option to the `./Configure` script. This
+    is recommended but not required. `egd` is supported at 3.0 but cannot be
+    used if FIPS is selected.
+  * The TNS/E platform does not support hardware randomization, so specify
+    the `--with-rand-seed=egd` option to the `./Configure` script.
```

Comment:
Please don't indent normal paragraphs by one byte. (If you're in doubt about our habits, please peek into INSTALL.md and CHANGES.md)

```suggestion
In addition to the requirements and instructions listed
in [INSTALL.md](INSTALL.md), the following are required as well:

 * The TNS/X platform supports hardware randomization. Specify
   the `--with-rand-seed=rdcpu` option to the `./Configure` script. This
   is recommended but not required. `egd` is supported at 3.0 but cannot be
   used if FIPS is selected.
 * The TNS/E platform does not support hardware randomization, so specify
   the `--with-rand-seed=egd` option to the `./Configure` script.
```

Reply:
Understood. I used the NOTES-VMS.md as a model. Probably a bad idea in hindsight.

Reply:
Using VMS as a model is always a bad idea. ;-)

Reply:
Hmmm...  I'll remember that remark, @mspncp :wink: